doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -72,4 +72,11 @@ Access GeoServer front page
 ---------------------------
 
 * After a few seconds, GeoServer should be accessible at: `<http://localhost:8080/geoserver>`_
-* The default ``admin`` password is ``geoserver``.
\ No newline at end of file
+* The default ``admin`` password is ``geoserver``.
+
+Run GeoServer from Intellij on Windows
+---------------------------
+
+#. Add bash to your Windows environment path and restart Intellij.  
+#. Rebuild the security-tests project in the security module.  Right click on the security-tests project and click Rebuild.  Otherwise,  there will be symbol errors such as "cannot find symbol AbstractUserGroupServiceTest".
```

Comment:
This one I don't remember seeing... the symbol errors complaining about "AbstractUserGroupServiceTest", were they in the same module? 
I'd change the above statement to make it conditional "if you get the cannot find symbol errors ... then ..."